stdlib/private/CMakeLists.txt

```diff
@@ -23,8 +23,8 @@ if(SWIFT_BUILD_SDK_OVERLAY)
 
   if(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
     add_subdirectory(StdlibUnittestFoundationExtras)
-    if (SWIFT_INCLUDE_TESTS)
-      add_subdirectory(SwiftReflectionTest)
-    endif()
+  endif()
+  if (SWIFT_INCLUDE_TESTS)
+    add_subdirectory(SwiftReflectionTest)
```

Comment:
This is causing SwiftReflectionTest.swift to be built on Windows, which fails due to missing definitions. Can you try the condition `SWIFT_INCLUDE_TESTS AND NOT CMAKE_SYSTEM_NAME STREQUAL "Windows"`, with an explanatory comment?

Reply:
Ok! I saw this as well, I was just waiting for the tests to end. Should I push or should I wait for the Linux test?

Reply:
I'll leave it up to you. The Linux test should be done soon. Feel free to IM me when you'd like me to kick off another round of testing.